README.md

```diff
@@ -4,13 +4,25 @@ Native implementation of Deep Learning models for GPU backends (mxnet, Caffe, Te
 
 ![architecture](./architecture/overview.png "Deep Water High-Level Architecture")
 
-## Requirements
-
-* SWIG: http://www.swig.org/
-
-* A C++ compiler with C++11 support
-
-`make pkg` will generate a `jar` file including native code.
+## Build Requirements
+
+1. A C++ compiler with C++11 support
+1. [SWIG](http://www.swig.org/)
+1. [BLAS](http://www.netlib.org/blas/)
+1. [OpenCV](http://opencv.org)
```

Comment:
I think we can live without OpenCV for now - I'd disable that in the mxnet build config.